libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -298,7 +298,7 @@ class database_api
        *    was just canceled accidentally, in such case, the result orders' price may lower or equal to start_price,
        *    but orders' id greater than start_id
        */
-      vector<limit_order_object> get_account_limit_orders( const string& name_or_id,
+      vector<limit_order_object> get_account_limit_orders( const string& account_name_or_id,
```

Comment:
Need to update comment about this variable. Scroll up.

Reply:
thanks, updated at https://github.com/bitshares/bitshares-core/pull/1168/commits/bd5c842e914d98d9222be57da132cf92b4a0123c